run-tests.sh

```diff
@@ -54,7 +54,7 @@ fi
 export LC_TIME=en_US.UTF-8
 python -m check_manifest
 python -m sphinx.cmd.build -qnNW docs docs/_build/html
-eval "$(docker-services-cli up --db ${DB:-postgresql} --search ${SEARCH:-opensearch} --mq ${MQ:-redis} --env)"
+eval "$(docker-services-cli up --db ${DB:-postgresql} --search ${SEARCH:-opensearch} --mq ${MQ:-rabbitmq} --cache ${CACHE:-redis} --env)"
```

Comment:
For some reason (locally, at least), I needed Redis... Not sure if it's related to the communities/users/groups caching or something else entirely.